resources.go

```diff
@@ -176,6 +147,12 @@ func Provider() tfbridge.ProviderInfo {
 				"pulumi": ">=1.0.0,<2.0.0",
 			},
 		},
+		CSharp: &tfbridge.CSharpInfo{
+			PackageReferences: map[string]string{
+				"Pulumi":                       "1.5.0-*",
+				"System.Collections.Immutable": "1.6.0",
+			},
+		},
```

Comment:
Please adjust all package names to PascalCase and build a `Namespaces` map, see https://github.com/pulumi/pulumi-openstack/pull/70 for example